src/tests/ftest/container/multiple_delete.py

```diff
@@ -61,7 +61,7 @@ def test_multiple_container_delete(self):
         self.log.info("SCM = %d, SSD = %d", final_scm_fs, final_ssd_fs)
 
         self.log.info("Verifying SSD space is recovered")
-        self.check_pool_free_space(self.pool, expected_nvme=initial_ssd_fs)
+        self.assertTrue(self.check_pool_free_space(expected_nvme=initial_ssd_fs))
```

Comment:
(pylint-no-member) Instance of 'MultipleContainerDelete' has no 'check_pool_free_space' member